samples/subscriptions/createSubscription.js

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019-2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * This application demonstrates how to perform basic operations on
+ * subscriptions with the Google Cloud Pub/Sub API.
+ *
+ * For more information, see the README.md under /pubsub and the documentation
+ * at https://cloud.google.com/pubsub/docs.
+ */
+
+'use strict';
+
+// Creates a new subscription.
+function main(
+  topicName = 'YOUR_TOPIC_NAME',
+  subscriptionName = 'YOUR_SUBSCRIPTION_NAME'
+) {
+  // [START pubsub_create_pull_subscription]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const topicName = 'YOUR_TOPIC_NAME';
+  // const subscriptionName = 'YOUR_SUBSCRIPTION_NAME';
+
+  // Imports the Google Cloud client library
+  const {PubSub} = require('@google-cloud/pubsub');
+
+  // Creates a client; cache this for further use
+  const pubSubClient = new PubSub();
+
+  async function createSubscription() {
+    // Creates a new subscription
+    await pubSubClient.topic(topicName).createSubscription(subscriptionName);
+    console.log(`Subscription ${subscriptionName} created.`);
+  }
+
+  createSubscription();
+  // [END pubsub_create_pull_subscription]
+}
+
+const {sampleMain} = require('../common');
+sampleMain()
```

Comment:
I think you might be able to avoid the wrapper class with something like this:

```
yargs.command('create <topicName> <subscriptionName>', 'creates a new subscription', (yargs) => {
  yargs.positional('topicName', {
    number: true // just an example of how to configure positionals
  })
}, (opts) => {
  main(opts.topicName, opts.subscriptionName);
})
.example('my-topic worker-1')
```

:point_up: I think using this shorthand for defining a `command` and `handler`, with the `.positional` feature for defining types, you could probably get away with using yargs directly.

Reply:
I'm going ahead and removing the yargs usage. Only a few samples use numeric parameters, and it's simple enough to throw in a call to Number() to make sure they're the right type.